tests/std/tests/P0645R10_text_formatting_custom_formatting/test.cpp

```diff
@@ -49,8 +71,78 @@ constexpr void test_disabled_formatter_is_disabled() {
     static_assert(!is_move_assignable_v<F>);
 }
 
+template <class T, class charT>
+void test_custom_equiv_with_format(const charT* fmt_string, T&& val) {

```

Comment:
`fmt_string` could be `const`. (Also on 80.)

Reply:
it's test code, not worth it.